src/components/react-component-page.tsx

```diff
@@ -0,0 +1,240 @@
+          <ComponentPageNav
+            basePath={data.sitePage.path}
+            includeReact={data.sitePage.context.frontmatter.reactId}
+            includeRails={data.sitePage.context.frontmatter.railsIds}
+            includeFigma={data.sitePage.context.frontmatter.figmaId}
+            includeCSS={data.sitePage.context.frontmatter.cssId}
+            current="react"
+          />
+        </Box>
+        <Box sx={{display: 'flex', flexDirection: 'row-reverse', alignItems: 'start', gap: [null, 7, 8, 9]}}>
+          <Box
+            sx={{
+              width: 220,
+              flex: '0 0 auto',
+              position: 'sticky',
+              top: HEADER_HEIGHT + 24,
+              maxHeight: `calc(100vh - ${HEADER_HEIGHT}px - 24px)`,
+              display: ['none', null, 'block'],
+            }}
+          >
+            <Heading as="h3" sx={{fontSize: 1, display: 'inline-block', fontWeight: 'bold', pl: 3}} id="toc-heading">
+              On this page
+            </Heading>
+            <TableOfContents aria-labelledby="toc-heading" items={tableOfContents.items} />
+          </Box>
+          <Box sx={{minWidth: 0}}>
+            {/* @ts-ignore */}
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: ['column', null, null, null, 'row'],
+                justifyContent: 'space-between',
+                gap: 3,
+                mb: 4,
+              }}
+            >
+              <Box
+                as={'ul'}
+                sx={{
+                  display: 'flex',
+                  flexWrap: 'wrap',
+                  gap: 2,
+                  alignItems: 'center',
+                  m: 0,
+                  p: 0,
+                  paddingInline: 0,
+                  listStyle: 'none',
+                  '& > li': {
+                    display: 'flex',
+                  },
+                }}
+              >
+                <li>
+                  <Label size="large">@primer/react@{data.primerReactVersion.version}</Label>
+                </li>
+                <li>
+                  <StatusLabel status={sentenceCase(status)} />
+                </li>
+                <li>
+                  <AccessibilityLabel a11yReviewed={a11yReviewed} short={false} />
+                </li>
+              </Box>
+              {statuses.length > 1 &&
+                <Box
+                  as={'ul'}
+                  sx={{
+                    display: 'flex',
+                    gap: 3,
+                    alignItems: 'center',
+                    m: 0,
+                    p: 0,
+                    paddingInline: 0,
+                    listStyle: 'none',
+                    fontSize: 1,
+                    '& > li': {
+                      display: 'flex',
+                    },
+                  }}
+                >
+                  <StatusMenu currentStatus={status} statuses={statuses} parentPath={`${data.sitePage.path}/react`} />
```

Comment:
When I navigate to `/latest`, will I get the newest version (usually "draft"), or will I get the version that's higher precedence in the [statusOrder array](https://github.com/primer/design/blob/main/src/status-utils.js#L2)?